firmware/controllers/core/error_handling.cpp

```diff
@@ -71,10 +71,6 @@ void chDbgPanic3(const char *msg, const char * file, int line) {
 #if EFI_HD44780_LCD
 	lcdShowPanicMessage((char *) msg);
 #endif /* EFI_HD44780_LCD */
-
-	if (!main_loop_started) {
```

Comment:
is main_loop_started still needed after this change?

Reply:
yes because I'm about to use it again